plugins/flytekit-omegaconf/tests/test_dictconfig_transformer.py

```diff
@@ -93,6 +93,7 @@ def test_extract_type_and_value_maps_simple() -> None:
     "type_desc, expected_type",
     [
         ("builtins.int", int),
+        ("builtins.NoneType", NoneType),
```

Comment:
This covers the corner case that makes this test fail without the suggested change in `dictconfig_transformer.py`